php/antidot/composer.json

```diff
@@ -0,0 +1,30 @@
+{
+  "name": "antidot-fw/reactive-starter",
+  "description": "Reactive Antidot Framework Apllication Starter",
+  "type": "project",
+  "license": "BSD-2-Clause",
+  "authors": [
+    {
+      "name": "kpicaza"
+    }
+  ],
+  "require": {
+    "php": "^7.4|^8.0",
+    "ext-json": "*",
+    "antidot-fw/react-framework": "~1.0",
```

Comment:
should be on `X.Y.Z` to guarantee that we keep staying at `X.Y` (if no lock file)